sklearn/preprocessing/_discretization.py

```diff
@@ -28,8 +28,12 @@ class KBinsDiscretizer(TransformerMixin, BaseEstimator):
 
     Parameters
     ----------
-    n_bins : int or array-like of shape (n_features,), default=5
+    n_bins : {int, array-like (n_features,), dtype=integral, 'auto'}, default=5
```

Comment:
The curly brackets are used to list the admissible values of a parameter of a same type like different strings. It does not apply here.
```suggestion
    n_bins : int, 'auto' or array-like of shape (n_features,), dtype=integral, default=5
```

Reply:
if it's too long it's ok to break the line with a ``\``